components/tools/OmeroWeb/omeroweb/webgateway/urls_api.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+# Copyright (C) 2016 University of Dundee & Open Microscopy Environment.
+# All rights reserved.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+# Handles all 'api' urls without including '/webgateway/' in the url.
+
+from django.conf.urls import url, patterns
+from omeroweb.webgateway import views
+from omeroweb.webgateway.views import LoginView, jsonp
+from django.conf import settings
+import re
+
+versions = '|'.join([re.escape(v)
+                    for v in settings.API_VERSIONS])
+
+api_versions = url(r'^$', views.api_versions, name='api_versions')
+
+api_base = url(r'^v(?P<api_version>' + versions + ')/$',
```

Comment:
Done in https://github.com/openmicroscopy/openmicroscopy/pull/4708/commits/e260602058952292a18fb08938cc89fa01a9734e